Gems/PhysX/Code/Editor/EditorJointConfiguration.h

```diff
@@ -71,6 +71,28 @@ namespace PhysX
         float m_limitNegative = -45.0f;
     };
 
+    /// Pair (linear) limits for joints.
+    class EditorJointLimitLinearPairConfig : public EditorJointLimitBase
+    {
+    public:
+        static const float s_linearLimitMin;
+        static const float s_linearLimitMax;
+
+        bool IsLimited() const;
+        JointLimitProperties ToGameTimeConfig() const;
+
+        AZ_CLASS_ALLOCATOR(EditorJointLimitLinearPairConfig, AZ::SystemAllocator, 0);
+        AZ_TYPE_INFO(EditorJointLimitLinearPairConfig, "{20A3AE4C-1B92-4541-ACA7-5FA2BFDDEDC0}");
```

Comment:
uber nit: Normally class allocation and type info are the first thing in the class, then functions, last members. This is uber nit, feel free to ignore.

Reply:
Fixed, also reordered other members so functions preceded variables.